Core/scripts/dirac-install.py

```diff
@@ -2442,11 +2442,12 @@ def checkoutFromGit(moduleName, sourceURL, tagVersion, destinationDir=None):
 
   if exportRes:
     return S_ERROR("Error while exporting from git")
+
+  # replacing the code
   if os.path.exists(fDirName + '/' + moduleName):
     cmd = "ln -s %s/%s" % (codeRepo, moduleName)
   else:
-    cmd = "mv %s %s" % (codeRepo, moduleName)
-
+    cmd = "mv %s %s" % (fDirName, os.path.join(cliParams.targetPath, moduleName))
```

Comment:
@zmathe please check this one.

Reply:
it is okay for me. Both command is equivalent.

Reply:
I tried out and it works. I discovered another bug, which I just fixed.